graft/coreth/tests/warp/genesis.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
this is temporary, this PR needs to wait until a prior PR gets merged so I can change this.